### PR TITLE
added documentation for logging

### PR DESCRIPTION
**Pull request by xmairaa**
- added documentation for logging
- page size increased 640px to 1024px, for better view
- Little bit modifications on documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -343,50 +352,54 @@ <h3>
 
 <p><h2>Status Codes:</h2></p>
```
Comment (ValentinTyhonov) [inline index.html]: Every title in <h2> tag make in one style : or with ':', or with '.', or without punctuation marks at the end

File: index.html [hunk 2]
```diff
@@ -261,7 +270,7 @@ <h2>
 
 <p>Available REST resources for generate are:</p>
```
[the remark this answers is not in the document]
Comment (xmairaa) [reply index.html]: we have a story for document update, this changes we can consider in that story.

File: index.html [hunk 3]
```diff
@@ -236,32 +243,44 @@ <h3>
 
 <p>Gives:</p>
```
Comment (ValentinTyhonov) [inline index.html]: In title above ('same example but') add comma before but

File: index.html [hunk 4]
```diff
@@ -236,32 +243,44 @@ <h3>
...
+"source":{"domainId":"example.domain","host":"host","name":"name","serializer":{"groupId":"com.github.Ericsson","artifactId":"eiffel-remrem-semantics",<br/>
+"version":"0.3.1"},"uri":"http://java.sun.com/j2se/1.3/"},"security":{"sdm":{"authorIdentity":"test","encryptedDigest":"sample"}}},<br/>
+"data":{"outcome":{"conclusion":"SUCCESSFUL"},"persistentLogs":[{"name":"firstLog","uri":"http://myHost.com/firstLog"},<br/>
+{"name":"otherLog","uri":"isbn:0-486-27557-4"}],"customData":[]},<br/>
+"links":[{"type":"ACTIVITY_EXECUTION","target":"aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee1"}]}]</pre>
 
 <p><strong>For loading protocol jars other than eiffel-remrem-semantics :</strong></p>
 <pre><code> $ java -Djava.ext.dirs="/path/to/jars/" -jar generate-cli.jar -f eiffeljobfinished.json -t eiffeljobfinished -mp protocoltype
 </code></pre>
 <strong>Note: In the above example, protocol jar file must be present inside "/path/to/jars/" folder</strong>
```
Comment (ValentinTyhonov) [inline index.html]: 'In' in lowercase (and in other notes lowercase after semicolon too)

File: index.html [hunk 5]
```diff
@@ -281,20 +300,20 @@ <h2>
...
-<h3>Versions Endpoint:</h3>
-"/versions" endpoint is used to get the versions of messaging protocols along with the current version of generate in a json format.<br>
+<h3>Versions End point:</h3>
+"/versions" end point is used to get the versions of messaging protocols along with the current version of generate in a JSON format.<br>
 
-Ex:  http://localhost:8080/generate-service/versions/ <br>
+URL:  http://&lt;host&gt;:&lt;port&gt;/&lt;application name&gt;/versions/ <br>
 
-Result:
+Sample Result:
```
Comment (ValentinTyhonov) [inline index.html]: 'result' in lowercase

File: index.html [hunk 6]
```diff
@@ -321,14 +340,16 @@ <h3>Ldap configurations:</h3>
...
+<pre>curl -H "Content-Type: application/json" -X POST -d '{"msgParams": {"meta": {"type": "EiffelActivityFinishedEvent", "tags": ["tag1","tag2"], "source": {"domainId": "example.domain", "host": "host", "name": "name", "uri": "http://java.sun.com/j2se/1.3/", "gav": {"groupId": "com.github.Ericsson", "artifactId": "eiffel-remrem-semantics", "version": "0.2.9"}}, "security": {"sdm": {"authorIdentity": "test", "encryptedDigest": "sample"}}}}, "eventParams": {"data": {"outcome": {"conclusion": "SUCCESSFUL"}, "persistentLogs": [{"name": "firstLog", "uri": "http://myHost.com/firstLog"}, {"name": "otherLog", "uri": "isbn:0-486-27557-4"}]}, "links": [{"type": "ACTIVITY_EXECUTION", "target": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee1"}]}}' http://localhost:8080/generate-service/eiffelsemantics?msgType=eiffelactivityfinished</pre>
 
 <p><strong>NOTE:</strong> For eiffel-remrem-semantics inputEventType is not mandatory it will be added based on event type query, But from the input.json the eventType should be camelcase</p>
```
Comment (ValentinTyhonov) [inline index.html]: Add comma after mandatory
'But' in lowercase after comma

File: index.html [hunk 7]
```diff
@@ -341,52 +362,56 @@ <h3>
...
+                </tr>
+                <tr>
+                    <td>200</td>
+                    <td>OK</td>
+                    <td>Event sent successfully</td>
+                </tr>
+                <tr>
+                    <td>400</td>
+                    <td>Bad Request</td>
+                    <td>Malformed JSON (Missing braces etc..), client need to fix problem in event before
```
Comment (ValentinTyhonov) [inline index.html]: 'Missing braces' in lower case

File: index.html [hunk 8]
```diff
@@ -341,52 +362,56 @@ <h3>
...
+                    <td>Service Unavailable</td>
+                    <td>When the message protocol is invalid</td>
+
+                </tr>
+            </table>
+
+            <h2>Logging</h2>
+
+
+<p>REMReM Generate application logging is implemented using the logback-classic, and we required to configure the logback.xml file, <br/>
```
Comment (ValentinTyhonov) [inline index.html]: Remove comma at the end of line. Or also add dot if needed

**Comment by ValentinTyhonov** [comment]
Looks good for me

File: index.html [hunk 9]
```diff
@@ -31,23 +30,27 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
...
-<a id="the-releases" class="anchor" href="#the-releases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>The releases</h2>
+            <h2>
+                <a id="the-releases" class="anchor" href="#the-releases" aria-hidden="true"><span aria-hidden="true"
+                    class="octicon octicon-link"></span></a>The releases
+            </h2>
 
-<p>Binary is relased via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-generate">jitPack</a>.</p>
-<p><b>Download the latest version of CLI from </b><a onClick="getLatestVersion('cli','jar');">generate-cli.jar</a></p>
-<p><b>Download the latest version of Service from </b><a onClick="getLatestVersion('service','war');">generate-service.war</a></p>
+            <p>Binary is release via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-generate">jitPack</a>.</p>
```
Comment (xvinosi-github) [inline index.html]: released

File: index.html [hunk 10]
```diff
@@ -81,38 +84,42 @@ <h2>
...
  -d,--debug                  enable debug traces
  -f,--content_file           message content file
  -h,--help                   show help.
- -json,--json_content        json content
+ -json,--json_content        JSON content
  -mp,--messaging_protocol    name of messaging protocol to be used,
                              e.g. eiffel3, semantics
  -r,--response_file          file to store the response in, optional
- -t,--message_type           message type, mandatory if -f or -json
+ -t,--message_type           message type, mandatory if -f or -JSON
```
Comment (xvinosi-github) [inline index.html]: -json only not -JSON

File: index.html [hunk 11]
```diff
@@ -81,38 +84,42 @@ <h2>
...
 <pre>java -jar generate-cli.jar -h
 You passed help flag.
 usage: java -jar
  -d,--debug                  enable debug traces
  -f,--content_file           message content file
  -h,--help                   show help.
- -json,--json_content        json content
+ -json,--json_content        JSON content
  -mp,--messaging_protocol    name of messaging protocol to be used,
                              e.g. eiffel3, semantics
```
Comment (l-ohlen-ericsson) [inline index.html]: Is the protocol name not eiffelsemantics, not semantics?

Comment (xmairaa) [reply index.html]: yes Lars it is eiffelsemantics only, I will do modify

**Pull request merged**